src/System.Runtime.Serialization.Formatters/src/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "frameworks": {
+    "netstandard1.4": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1-rc3-24102-00",
+        "System.Collections": "4.0.10",
```

Comment:
What depends on System.Collections in this library? It is worth trying to clean-up the dependencies here.